### ISSUE TITLE
feat:(TikTok) Disable autoplay

### ISSUE THREAD

**Issue by Jeanfeels** [ReVanced/revanced-patches#3450]
### Feature description

I have no idea if this is possible, and it has yet to be suggested.
This feature ideally would be an ideal way to prevent videos from automatically playing, and/or automatically loading the next video in the feed. Alternatively, adding or modifying the autoplay delay between videos to be extremely long would be better than nothing if it is more feasible. 

### Motivation

-The 'endless play' barrage of content, while seemingly a core feature of tiktok, is absolutely crippling for many of us with sensory overload issues and is actively despised by enough people that this should be considered a worthwhile feature if it's even possible to patch in. It is an accessability feature that many of us would be incredibly grateful for.

-Videos loading immediately on app launch are a waste of mobile data or data limited wifi access. The in-app settings to address this are not adequate and do not function when connected to wifi, even if the network is rate limited with a metered data cap.

-It is frustrating and annoying to be watching a video and have it immediately load a completely unrelated video as if you'd swiped up after a short delay. While it may be argued that this is "the point" of tiktok, it would be nice to have a way to turn that off or set the timer to a much higher integer for a more relaxing experience with more control over the content watched. 

-The setting for muting the app on startup occasionally fails to work, even in the unpatched vanilla app. This feature would preclude this from being an issue, as a bonus benefit.

In summary:
This would be useful to anyone with a subset of anxiety, sensory issues, related disabilities, a restrictive data cap on a metered wifi connection (a very common situation especially outside of cities), or simply a strong dislike of having random videos shoved at them. It may turn out to be impossible, but on the off chance it isn't, the benefits to the wider community would be worth considering.

### Acknowledgements

- [X] I have checked all open and closed feature requests and this is not a duplicate
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
